README.md

```diff
@@ -74,6 +74,10 @@ Not too different to your usual workflow!
 7. This can be done multiple times and it will increment your pre-release package version!
 8. When you're happy with the changes, simply squash and merge the PR and release `main`!
 
+###### Tokens
+
+The `MARSHMALLOW_CI_PAT` has been created from the internal @marshmallow-ci GH account, it is due to expire on `9th December 2024` so will need regenerating once it does.
```

Comment:
this also needs adjusting to whatever the expiration date is on the new PAT that platform creates